backend/app/api/endpoints/wiki.py

```
@@ -79,6 +81,50 @@ def _verify_internal_token(
+    """Authenticate an internal caller and say which user it is, if any.
+
+    ``None`` means the fixed internal token was used, which is a trusted operator
+    rather than a person and is not scoped to one generation.
+    """
+    _verify_internal_token(authorization=authorization, db=db)
+    token = authorization[7:].strip()
+    if token == wiki_settings.INTERNAL_API_TOKEN:
+        return None
+    try:
+        return security.get_current_user_from_token(token, db)
+    except Exception:  # pragma: no cover - _verify_internal_token already accepted it
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)